template-compose/app/build.gradle.kts

```diff
@@ -34,18 +52,8 @@ android {
         }
     }
 
-    compileSdk = Versions.ANDROID_COMPILE_SDK
-    defaultConfig {
-        applicationId = "co.nimblehq.template.compose"
-        minSdk = Versions.ANDROID_MIN_SDK
-        targetSdk = Versions.ANDROID_TARGET_SDK
-        versionCode = getVersionCode()
-        versionName = Versions.ANDROID_VERSION_NAME
-        testInstrumentationRunner = "androidx.test.runner.AndroidJUnitRunner"
-    }
-
     buildTypes {
-        getByName(BuildTypes.RELEASE) {
+        release {
```

Comment:
Why do we need to update this? 🤔

Reply:
@lydiasama `release {` is a new supported block from Gradle 8's DSL. We should use any built-in features instead of manual configurations 🤓